R/dm.R

```diff
@@ -481,6 +496,18 @@ print.dm <- function(x, ...) {
   invisible(x)
 }
 
+print_zoomed <- function(x) {
```

Comment:
Ideally we'd implement `format()` and forward `print()` to it.

Reply:
Not sure how the `format()`-method(s) should look like. I will postpone it, unless you describe how it should look, if that's ok.